server/src/main/java/org/opensearch/index/codec/CodecService.java

```diff
@@ -59,27 +61,37 @@ public class CodecService {
 
     private final Map<String, Codec> codecs;
 
-    public static final String DEFAULT_CODEC = "default";
-    public static final String BEST_COMPRESSION_CODEC = "best_compression";
+    public static final String LZ4_CODEC = "lz4";
```

Comment:
Can leave in the default and best_compression initializations here to reduce code change.